CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.19.0] - 2022-03-21
+
+### Changed
+
+- Return events from blocks skipped over during Finalization, too ([#473](git@github.com:paritytech/subxt/pull/473))
```

Comment:
This seems like a mismatch generated by `./scripts/generate_changelog.sh`.

It seems that the script inspects `git config --get remote.origin.url`, which is not always expected to be a browser URL.  Will do a quick fix for it in the meanwhile.

Reply:
oooh indeed; I didn't pay attention to those URLs :)